examples/src/main/scala/examples/hybrid/history/HybridSyncInfo.scala

```diff
@@ -33,7 +33,7 @@ case class HybridSyncInfo(answer: Boolean,
 }
 
 object HybridSyncInfo {
-  val MaxLastPowBlocks = 50 //don't make it more than 127 without changing serialization!
+  val MaxLastPowBlocks: Int = 50 //don't make it more than 127 without changing serialization! (use Byte as type?)
```

Comment:
@ceilican 
should this be set to type Byte as mentioned?

Reply:
I think it would be a good idea.